src/user/index.js

```diff
@@ -1,5 +1,4 @@
 export * from './user.action';
 export * from './user.reducer';
 export * from './user.type';
-
```

Comment:
If it does not really need, maybe we can revert this line.

Reply:
:point_up: